### ISSUE TITLE
I would very much love to contribute

### ISSUE THREAD

**Issue by ghost** [AmbitionFramework/ambition#15]
Hi, you didn't have any kind of CONTRIBUTING.md or such in your working tree so this is my only known way of reaching out to the developers.

I am an individual who would very much love to see vala grow, and I personally believe the best market for Vala is exactly what ambition is targeting: frameworks that already exist in the enterprise field in languages like Java and C#. It would be but my pleasure to help you out with this. The issues that exist in this repo are rather old, and breathing new life to this framework would make it more known. 

So my question is: how may I help with development?

**Comment by nmelnick** [comment]
Hi, @apachejuice ! That's wild, I literally just [dusted this project off](https://github.com/AmbitionFramework/ambition/tree/rethink-refresh) last night to pull it into a more modern build environment and start in on the 0.2 branch again. Let me think this over tonight as I finish up a couple of thoughts and figure out how to open up the development process to a larger group. I'd absolutely love the help.

**Comment by apachejuice** [comment]
> Hi, @apachejuice ! That's wild, I literally just [dusted this project off](https://github.com/AmbitionFramework/ambition/tree/rethink-refresh) last night to pull it into a more modern build environment and start in on the 0.2 branch again.

yeah, thats the reason i reached out :) i did notice the project a couple weeks ago and was like "ugh its dead such a shame". Glad you picked it up again!

also you can call me Sara if you wish.
